### ISSUE TITLE
Action failing in pipeline in version 1.0.8

### ISSUE THREAD

**Issue by bertoverflow** [jpb06/coverage-badges-action#11]
Hello and thanks for your awesome work on this github action :)!

We are using your action in our pipelines to create coverage badges.

However, the version released yesterday (v1.0.8) does not work in our pipelines.

Running the action produces the following error:
```
Run jpb06/coverage-badges-action@latest
node:internal/modules/cjs/loader:1051
  throw err;
  ^

Error: Cannot find module '../node_modules/effect-errors/cjs/index.js'
Require stack:
- /home/runner/work/_actions/jpb06/coverage-badges-action/latest/dist/index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1048:15)
    at Module._load (node:internal/modules/cjs/loader:901:27)
    at Module.require (node:internal/modules/cjs/loader:1115:19)
    at require (node:internal/modules/helpers:130:18)
    at /home/runner/work/_actions/jpb06/coverage-badges-action/latest/dist/index.js:1:13693
    at /home/runner/work/_actions/jpb06/coverage-badges-action/latest/dist/index.js:1:35
    at Object.<anonymous> (/home/runner/work/_actions/jpb06/coverage-badges-action/latest/dist/index.js:1:13948)
    at Module._compile (node:internal/modules/cjs/loader:1241:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1295:10)
    at Module.load (node:internal/modules/cjs/loader:1091:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/runner/work/_actions/jpb06/coverage-badges-action/latest/dist/index.js'
  ]
}

Node.js v20.8.1
```

This is how we integrate the action in our pipeline (shortened):
```
  code-coverage:'
    runs-on: ubuntu-latest

    steps:
      - uses: actions/checkout@v4
      - uses: actions/setup-node@v4
        with:
          node-version: '18.18'
      - run: yarn install --frozen-lockfile

      - run: yarn test:coverage

      - name: Generating coverage badges
        uses: jpb06/coverage-badges-action@v1.0.7
        with:
          coverage-summary-path: ./coverage/coverage-summary.json
          branches: main
          commit-message: 'chore: update badges for code coverage'
```

Version 1.0.7 was working fine. Could this also be an issue on our side? Do you need any further information.

Thanks a lot!

**Comment by jpb06** [comment]
Hello!

Sorry about that, I broke something. Looks like some issue with the bundling, as I moved to esbuild.
Action works fine when not published though. Taking a look!

https://github.com/jpb06/coverage-badges-action/pull/12

**Comment by jpb06** [comment]
Resolved as [1.0.9](https://github.com/jpb06/coverage-badges-action/releases/tag/v1.0.9). 

Please let me know how it rolls for you.
